std_msgs/Header header
sparse_dynamic_calibration/PoseSim3 pose
